R/process_selectors.R

```diff
@@ -159,7 +177,7 @@ process_formula_selectors <- function(data, ..., env = caller_env(),
 
 #' @name process_selectors
 #' @export
-fill_formula_selectors <- function(data, ..., env = caller_env()) {
+fill_formula_selectors.data.frame <- function(data, ..., env = caller_env()) {
```

Comment:
Do these generics not need to have separate documentation instead of all using the `process_selectors` doc? 
The parameters are different for all of them

Reply:
To me, the docs look fine; they depend on the file grouping, and it seems that this file contains only functions related to selector processing. The parameters defined here seems to be consistent for the page

Reply:
That's a good point. They are all related, and these will mostly be used by other devs. I think they can be documented together.